components/Modal/CreateImageUpload.js

```diff
@@ -311,23 +337,111 @@ class CreateImageUploadModal extends React.Component {
       uploadService
     } = this.state;
 
-    const providerCheckbox = (provider, displayName) => (
-      <FormGroup
-        label={this.props.intl.formatMessage({ id: `${provider}-form`, defaultMessage: "Upload image" })}
-        fieldId="provider-checkbox"
-      >
-        <Checkbox
-          value={provider}
-          isChecked={this.state.uploadService === provider}
-          onChange={this.handleUploadService}
-          label={this.props.intl.formatMessage({
-            id: `${provider}-checkbox`,
-            defaultMessage: `Upload to ${displayName}`
-          })}
-          id={`${provider}-checkbox`}
-          aria-labelledby="provider-checkbox"
-        />
-      </FormGroup>
+    const awsProviderCheckbox = (
+      <div className="pf-c-form__group">
+        <div className="pf-c-form__label pf-m-no-padding-top pf-l-flex pf-u-display-flex pf-m-justify-content-flex-start pf-m-nowrap">
+          <span className="pf-l-flex__item pf-c-form__label-text">
+            <FormattedMessage defaultMessage="Upload image" />
+          </span>
+          <Popover
+            id="aws-provider-popover"
+            bodyContent={
+              <TextContent>
+                <p>
+                  <FormattedMessage
+                    defaultMessage="
+                      Image Builder can upload images you create to an {bucket} in AWS and then import them into EC2. When the image build is complete
+                      and the upload action is successful, the image file is available in the AMI section of EC2. Most of the values required to upload
+                      the image can be found in the {console}.
+                    "
+                    values={{
+                      bucket: "S3 bucket",
+                      console: <a href="https://console.aws.amazon.com/console/home">AWS Management Console</a>
```

Comment:
When I click this link, nothing happens. However if I right-click and select to open the link in a new tab, it does work. I would expect in this context that external links to automatically open in a new tab. 

When including external links, we should include the icon `<ExternalLinkSquareAltIcon />` at the end of the text. The [PF react button component](https://www.patternfly.org/v4/documentation/react/components/button#variations) includes an example of how this looks, but if you were going to use the react component, I think we would also need the `isInline` prop so that the link displays proper spacing when inline with text.

Reply:
This icon has been added. Is this the desired alignment?

![linkIconAlignment](https://user-images.githubusercontent.com/11712857/83778716-5ed67780-a68b-11ea-8afc-064bd6ddc619.png)

Reply:
I see the same alignment when I build locally. When I compare the runtime html to the example on [PF.org](https://www.patternfly.org/v4/documentation/react/components/button), I see that the `<svg>` in the PF example includes `vertical-align` which results in a lower position:
`<svg style="vertical-align: -0.125em;" fill="currentColor" height="1em" width="1em" viewBox="0 0 448 512" aria-hidden="true" role="img">`

But it isn't obvious to me why our `<svg>` is missing that `vertical-align` property. As a short term fix, we could add the same styles to our css file, for example:
```
.pf-icon svg {
  vertical-align: -0.125em;
}
```
and apply `.pf-icon` to the this link.

Also, I noticed that this is a `<button>` element, but it should be an `<a>` element since it has an `href` attribute.